pages/linux/po4a.md

```diff
@@ -0,0 +1,8 @@
+# po4a
+
+> Update both the PO files and translated documents in one shot.
```

Comment:
```suggestion
> Update both PO files and translated documents
```

Reply:
I don't think we should leave the `the`. Otherwise it sounds like there are two PO files and you update both of them.